### PR TITLE
Tutorial beginners

### PR DESCRIPTION
**Pull request by robsontenorio**
Feel free to modify!

### CODE REVIEW

File: docs/getting-started-tutorial-beginners.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+## The Models
+
+One user can publish many posts, and each post has many comments from anonymous users.
+
+<div align="center">
+  <img src="/docs/assets/tutorial/model.png">  
+  <p><small>Diagram</small></p>
+</div>
+
+This is pure Laravel. After creating models and migrations remember to run migrate command:
```
Comment (olivernybroe) [inline docs/getting-started-tutorial-beginners.md]:
```suggestion
This first part is pure Laravel, we will add GraphQL part afterwards. Remember to run migration command after  creating models and migrations:
```

File: docs/getting-started-tutorial-beginners.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+    public function down()
+    {
+        Schema::dropIfExists('comments');
+    }
+}
+```
+
+## The Magic
+
+Let's edit `routes/schema.graphql` and define our blog schema, based on Eloquent Models we have created. This is mostly like a pure GraphQL schema definition, but with some Lighthouse sorcery.
```
Comment (olivernybroe) [inline docs/getting-started-tutorial-beginners.md]:
```suggestion
Let's edit `routes/schema.graphql` and define our blog schema, based on the Eloquent Models we created. This is pure GraphQL schema definition (SDL), but with some Lighthouse sorcery through directives. 
```

File: docs/getting-started-tutorial-beginners.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+}
+
+```
+
+
+In this example:
+
+
+- Note that `type` name is the same as a Eloquent Model
+- Directives add some behavior in your schema
```
Comment (olivernybroe) [inline docs/getting-started-tutorial-beginners.md]: Maybe make all of these link to the official documentation also.

File: docs/getting-started-tutorial-beginners.md [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+  - `@hasMany` / `@belongsTo` express eloquent relationships 
+  - `@all` fetch all eloquent models and return the collection
+  - `@find` find a model based on the arguments provided
+  - `@eq` places an equal operator on a eloquent query.  
+
```
Comment (olivernybroe) [inline docs/getting-started-tutorial-beginners.md]: Maybe add something like "There are tons of other directives in Ligthouse, you can read more about them here".

**Comment by spawnia** [comment]
Hey @robsontenorio 

Digging through at the moment. Really like what i see so far, just adding that extra
bit of polish.

**Comment by robsontenorio** [comment]
@spawnia do it ;)

**Comment by spawnia** [comment]
@robsontenorio Great stuff. Was too short on time lately, going to iterate on it later.

**Pull request merged**